library/common/http/client.h

```diff
@@ -213,7 +258,20 @@ class Client : public Logger::Loggable<Logger::Id::http> {
 
   using DirectStreamWrapperPtr = std::unique_ptr<DirectStreamWrapper>;
 
-  DirectStreamSharedPtr getStream(envoy_stream_t stream_handle);
+  enum Permissions {
```

Comment:
nit: I would probably call this `GetStreamFilters` or something like that so it's more specific to the function and also "permissions" has ACL connotations.